index.html

```diff
@@ -5,6 +5,9 @@
     <title>Mini Weather App</title>
     <link rel="stylesheet" href="css/style.css">
```

Comment:
Make sure your custom CSS is linked after any libraries that you'll be using. This should be loaded after bootstrap and the flag css files.